src/styles/card.css

```diff
@@ -0,0 +1,85 @@
+.card {
+  padding: 32px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: auto auto 40px;
+}
+
+.card__stuff-container {
+  color: #060b35;
+}
+
+.card__product-name {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__rating-line {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  margin-bottom: 24px;
+}
+
+.card__rating-rewiews {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-line {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__price-title {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+  font-weight: 400;
+}
+
+.card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__button {
+  display: block;
+  text-align: center;
+  font-size: 14px;
+  line-height: 40px;
+  font-weight: 700;
+  text-decoration: none;
+  color: #fff;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  transition: 0.3s;
+}
```

Comment:
divide your properties by rules, https://codeguide.academy/html-css.html#css-order

Reply:
#00acdc and , also we use a lot colors, maybe batter to add them to variables